decidim-meetings/app/views/decidim/meetings/directory/meetings/index.html.erb

```diff
@@ -1,20 +1,14 @@
-<% add_decidim_page_title(t(".meetings")) %>
-
-<div class="wrapper">
-  <div class="row">
-    <h1 class="section-heading" id="meetings-count">
-      <%= render partial: "decidim/meetings/meetings/count" %>
-    </h1>
-
-    <%= cell "decidim/meetings/meetings_map", search.result %>
-
-    <div class="columns mediumlarge-4 large-3">
-      <div class="card card--secondary show-for-mediumlarge">
-          <%= render partial: "filters" %>
-      </div>
-    </div>
-    <div id="meetings" class="columns mediumlarge-8 large-9">
-      <%= render partial: "meetings" %>
-    </div>
-  </div>
-</div>
+<%= render partial: "decidim/meetings/shared/index", locals: {
+  display_map: true,
+  display_announcement: false,
```

Comment:
Accessibility violations on the meetings directory page:
![Meetings directory a11y](https://user-images.githubusercontent.com/864340/209191744-94ff7858-404a-4a9e-8a81-d6db796a4755.png)